src/main/java/org/candlepin/sync/Exporter.java

```diff
@@ -118,6 +119,7 @@ public Exporter(ConsumerTypeCurator consumerTypeCurator, OwnerCurator ownerCurat
         CdnCurator cdnCurator,
         CdnExporter cdnExporter,
         SyncUtils syncUtils,
+        @Named("SyncUtilsObjectMapper") ObjectMapper mapper,
```

Comment:
This should be named something more specific, since the mapper isn't associated with the sync utility, but with the sync (import/export) operation itself. SyncObjectMapper, ImportObjectMapper, ManifestObjectMapper, etc..